src/hotspot/share/gc/g1/g1ConcurrentMark.cpp

```diff
@@ -2263,7 +2269,7 @@ bool G1CMTask::regular_clock_call() {
     return false;
   }
 
-  double curr_time_ms = os::elapsedVTime() * 1000.0;
+  jlong curr_time_ns = os::current_thread_cpu_time();
```

Comment:
pre-existing: I wonder why this is here, rather than a few lines later, near where it's used.

Reply:
I do not know either, and found it confusing too. I fixed it.